### PR TITLE
updates: Completed login page

### PR DESCRIPTION
**Pull request by GarvG**
Completed my login page please review my PR

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.form-container
+{
+    background: var(--white);
+    width: 20rem;
+    height: 22rem;
+    position: relative;
+    text-align: center;
+    padding: 2rem 0;
+    margin: auto;
+    box-shadow: 0 0 20px 0px rgb(0 0 0 / 10%);
```
Comment (zoef-yt) [inline login.css]: use var for color.

File: styles.css [hunk 2]
```diff
@@ -36,9 +36,8 @@
     background: var(--black);
     color:var(--white);
     padding: 0.5rem 2.2rem;
-    margin: 2.2rem 0;
+    /* margin: 2.2rem 0; */
```
Comment (zoef-yt) [inline styles.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: login.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    cursor: pointer;
+    width: 6.2rem;
+    display: inline-block;
+    font-size: 2rem;
+}
+.btn{
+    display: inline-block;
+    background: var(--black);
+    color:var(--white);
+    padding: 0.5rem 2.2rem;
```
Comment (Shubhamkr2610) [inline login.css:53]: use pixel for less than 20px values

File: login.css (hunk 1 above)
Comment (Shubhamkr2610) [inline login.css]: to maintain consistency , use color variable name

File: styles.css (hunk 2 above)
Comment (Shubhamkr2610) [inline styles.css]: remove unused code

Comment (GarvG) [reply styles.css]: removed it

File: login.css (hunk 1 above)
Comment (GarvG) [reply login.css]: done

Comment (GarvG) [reply login.css]: done with changes

File: styles.css (hunk 2 above)
Comment (GarvG) [reply styles.css]: done with changes

File: login.css (hunk 3 above)
Comment (GarvG) [reply login.css:53]: i have followed the convention of using pixels for less than 10 in the project

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good me . You can merge now

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**